File: tfm-sgpfe-simulator-rneves/service_simulator/lib/modules/common/domain/entities/service_post_office.dart
import 'package:equatable/equatable.dart';
import 'package:meta/meta.dart';

class ServicePostOffice extends Equatable {
  final int id;
  final double latitude;
  final double longitude;
  final String description;
  final int serviceId;
  final String address;

  ServicePostOffice(
      {@required this.id,
      @required this.latitude,
      @required this.longitude,
      @required this.serviceId,
      @required this.description,
      @required this.address});

  @override
  List<Object> get props =>
      [id, latitude, longitude, serviceId, description, address];
}

File: tfm-sgpfe-rneves/server/src/main/kotlin/isel/meic/tfm/fei/entities/ServicePostOffice.kt
package isel.meic.tfm.fei.entities

/**
 * Instituto Superior de Engenharia de Lisboa
 * Project: Smart Queue
 * @author Ronilton Neves - 39643
 * @mentor: Paulo Pereira
 *
 * <Description>
 */
data class ServicePostOffice(
    val id: Int,
    val latitude: Double,
    val longitude: Double,
    val description: String,
    val serviceId: Int,
    val address: String
)